plugins/woocommerce-admin/client/marketing/overview-multichannel/DiscoverTools/DiscoverTools.tsx

```diff
@@ -133,8 +133,75 @@ export const DiscoverTools = () => {
 
 					return (
 						<>
-							<CardDivider />
-							{ renderPluginCardBodies( subcategoryPlugins ) }
+							{ subcategoryPlugins.map( ( el ) => {
```

Comment:
Nitpicky, but could we update `el` to `plugin`?  At first glance it feels like a dom element or something that's not a plugin.

Reply:
Done in [`ccc4dd4` (#35542)](https://github.com/woocommerce/woocommerce/pull/35542/commits/ccc4dd4b09e7fe7682dc638d561d909bdf66e9fa).

> At first glance it feels like a dom element or something that's not a plugin.

Kind of a personal preference or practice, I am used to writing array methods like `array.map( (el, idx, arr) => {} )`, where `el` means array element. I remember I learned these naming from MDN long time ago.